docs/reference/commandline/ps.md

```diff
@@ -27,6 +27,7 @@ Options:
                         - since=(<container-name>|<container-id>)
                         - ancestor=(<image-name>[:tag]|<image-id>|<image@digest>)
                           containers created from an image or a descendant.
+                        - is-task=(true|false|any)
```

Comment:
just by looking at "is-task", a user can deduct it's a boolean. I don't think just "task" is very descriptive, consider:

```
--filter task=true
--filter ancestor=true
```

First checks for a boolean value, second for an ancestor image named "true"